tests/installation/ntp_configuration.pm

```diff
@@ -0,0 +1,25 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: This test module launches the installation from
+#          the installation settings page.
+
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use testapi;
+use Test::Assert ':all';
+
+sub run {
+    my $ntp_settings = $testapi::distri->get_ntp_config();
+    diag '*****' . $ntp_settings;
+    assert_not_matches(qr/suse\.pool\.ntp\.org/, $ntp_settings, "NTP setting is not correct");
```

Comment:
Should the module run for MicroOS as well? If so there is opensuse pool server used.